nfc/NfcCx/Cx/NfcCxESE.h

```diff
@@ -0,0 +1,211 @@
+/*++
+
+Copyright (c) Microsoft Corporation.  All Rights Reserved
+
+Module Name:
+
+    NfcCxESE.h
+
```

Comment:
Majority of the code in NfcCxESE.cpp/.h is copied from NfcCxSE.cpp. I'm not sure what benefits are we getting from having all the code routines for SE/eSE in separated files, even if the logic behind is roughly the same. But on the downside, Class Extension code complexity is now multiplied; component's  scalability is more limited; bugs analysis & codebase maintaince becomes more time-consuming; etc. I'd be happy to work with you on refactoring the design to use more abstract reusable logic, reducing the size at the same time.